.github/workflows/build-docs.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     # This prevents this workflow from running on a fork.
     # To test this workflow on a fork, uncomment the following line.
     if: github.repository == 'nilearn/nilearn'
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Though I don't think that's related to the doc build failure, it should be updated. Probably best to use `ubuntu-latest`

Reply:
yup I was pretty sure this was not gonna fix it but felt like it would not hurt

You are correct: will switch to `latest`